pkgs/tools/security/pass/wofi-pass.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pass-wayland
+, coreutils
+, gnugrep
+, libnotify
+, pwgen
+, findutils
+, makeWrapper
+, wl-clipboard
+, wtype
+, wofi
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wofi-pass";
+  version = "23.1.2";
+
+  src = fetchFromGitHub {
+    owner = "schmidtandreas";
+    repo = "wofi-pass";
+    rev = "v${version}";
+    sha256 = "sha256-Z1a+nfnL6NXhufpCU7VIkwuifoS88Pf9qI8GCuP/Zqc=";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -a wofi-pass $out/bin/wofi-pass
+
+    mkdir -p $out/share/doc/wofi-pass/
+    cp -a wofi-pass.conf $out/share/doc/wofi-pass/config.example
+  '';
+
+  wrapperPath = lib.makeBinPath [
+    coreutils
+    findutils
+    gnugrep
+    libnotify
+    (pass-wayland.withExtensions (ext: [ ext.pass-otp ]))
```

Comment:
How does that compose if I pass a pass-wayland with existing extensions? It seems like that you should not impose extensions here or you should add a parameters for extensions.

Reply:
This should be fixed with the `WithExtensions` parameter added in the latest commit.